src/Network/Network/Network.csproj

```diff
@@ -41,9 +42,6 @@
     <Folder Include="RouteServerPeer\" />
   </ItemGroup>
   
-  <ItemGroup>
-    <ProjectReference Include="..\Network.Management.Sdk\Network.Management.Sdk.csproj" />
```

Comment:
Why is this removed?

Reply:
These are instructions by Mikhail to enable network sdk.